dart/lib/src/protocol/sentry_response.dart

```diff
@@ -0,0 +1,78 @@
+import 'package:meta/meta.dart';
+
+/// The response interface contains information on a HTTP request related to the event.
+/// This is an experimental feature. It might be removed at any time.
+@experimental
+@immutable
+class SentryResponse {
```

Comment:
Let's add this type builtin in the `Contexts` now.
It's gonna be part of `_defaultFields`, `copyWith` etc